services/minespace-web/src/components/dashboard/mine/overview/Overview.js

```diff
@@ -10,6 +10,7 @@ import {
   getCommodityOptionHash,
 } from "@common/selectors/staticContentSelectors";
 import { getTransformedMineTypes } from "@common/selectors/mineSelectors";
+import WorkerInfoEmployee from "./WorkerInfoEmployee";
 import { getUserInfo } from "@/selectors/authenticationSelectors";
```

Comment:
Can you make this an alias import? '@/components/dashboard/mine/WorkerInforEmployee`

Reply:
For sure, this is change now.